definitions/redux_v3.x.x/flow_>=v0.23.x/redux_v3.x.x.js

```diff
@@ -0,0 +1,29 @@
+declare module 'redux' {
+  declare type State = any;
+  declare type Action = Object;
```

Comment:
@gaearon, @jeffmo  if I understand correctly, various types of actions can be dispatched as an async action. So the interface of dispatch will be always `Action | AsyncAction`, and as `AsyncAction = any` it will become same as `Action | any` or simply `any`. So there's no reason at all to specify interface for actions. I think it would make more sense, to make the store generic:

For example:

```
declare type Store<S, A> = {
        dispatch: Dispatch<A>,
        getState: () => S,
        subscribe: (listener:() => void) => () => void,
        replaceReducer: (reducer:Reducer<S>) => void
};
```